workloads/statefulset/README.md

```diff
@@ -0,0 +1,43 @@
+# StatefulSet Workload
+
+oam-kubernetes-runtime also supports K8s native resource, this section will introduce how to use K8s StatefulSet.
+
+## Update RBAC for OAM AppConfig Controller
+
+Make sure your cluster-role has following rules:
+
+```yaml
```

Comment:
Do you plan to put this `ClusterRole` yaml to role.yaml?

Reply:
That need to change crossplane repo which is out of this PR's scope.